addon/has-ember-version.ts

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
```

Comment:
Stolen from `@ember/test-helpers`. Feels like this should be in a library somewhere - perhaps it is.